src/main/java/at/ac/tuwien/kr/alpha/grounder/atoms/RuleAtom.java

```diff
@@ -57,11 +57,8 @@ private RuleAtom(List<ConstantTerm<String>> terms) {
 		this.terms = terms;
 	}
 
-	public RuleAtom(NonGroundRule nonGroundRule, Substitution substitution) {
-		this(Arrays.asList(
-			getInstance(Integer.toString(nonGroundRule.getRuleId())),
-			getInstance(substitution.toString())
-		));
+	public RuleAtom(InternalRule nonGroundRule, Substitution substitution) {
+		this(Arrays.asList(getInstance(Integer.toString(nonGroundRule.getRuleId())), getInstance(substitution.toString())));
```

Comment:
Please do not destroy formatting that has obviously been chosen to improve readability.